pkgs/by-name/li/liba2i/package.nix

```
@@ -0,0 +1,39 @@
+{ stdenv
+, fetchgit
+, lib
+, pkg-config
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "liba2i";
+  version = "unstable-at-2024-01-13";
```

```suggestion
  version = "0-unstable-2024-01-13";
```

According to most recent documentation updates (missing link - insert here later)

---

Are you sure about the `0-` prefix?

---

yes:

https://github.com/NixOS/nixpkgs/pull/280312/files

> Example: Given a project that has no tags / released versions at all, or applies versionless tags like `latest` or `YYYY-MM-DD-Build`, a commit authored on March 15, 2022 would have `version = "0-unstable-2022-03-15"`.

---

Fixed.